### ISSUE TITLE
The iface zif_abapgit_repo should declare the method get_dot_abapgit

### ISSUE THREAD

**Issue by jfilak** [abapGit/abapGit#5462]
All kind of repos should be able to return the configuration object .abapgit. 

After the PR #5329 my scripts got broken because I was doing this:

```abap
      data(repo) = zcl_abapgit_repo_srv=>get_instance( )->new_online(
        iv_url = git_repo_url
        iv_branch_name = branch
        iv_package = abap_package
        iv_folder_logic = folder_logic
        iv_ign_subpkg = ignore_subpackages
        iv_main_lang_only = main_lang_only
      ).

      repo->get_dot_abapgit( )->set_starting_folder( starting_folder ).
```

**Comment by mbtools** [comment]
+1

**Comment by sbcgua** [comment]
I think it makes sense to move this and some other methods in. I moved just minimal necessities within https://github.com/abapGit/abapGit/pull/5329.

**Comment by mbtools** [comment]
closed by #5466
